kartothek/utils/migration_helpers.py

```diff
@@ -212,13 +210,11 @@ def _assemble_warning_message(parameter: str, message: str, func_name: str) -> s
         + func_name
     )
 
-
+# Adds neccessary meta info making multiple instance of these deprecation decorators stackable while the parameter
+# check before runtime stays intact.
```

Comment:
Not quite sure if `kartothek` documents private functions, but I would even turn this into a docstring instead of a comment.

Reply:
Done